addons/mail/models/mail_activity.py

```diff
@@ -757,6 +759,19 @@ def _search_activity_type_id(self, operator, operand):
     def _search_activity_summary(self, operator, operand):
         return [('activity_ids.summary', operator, operand)]
 
+    @api.depends('activity_ids.date_deadline')
+    @api.depends_context('uid')
+    def _compute_my_activity_date_deadline(self):
+        self.my_activity_date_deadline = next((
```

Comment:
That's strange as in multi mode, you link the first found activity to all records. Shouldn't you do it in a loop ?

Reply:
I second that, you definitively need a loop on `self`.